Lib/glyphsLib/builder/instances.py

```diff
@@ -71,9 +71,21 @@ def _to_designspace_varfont(self, instance):
     from fontTools.ufoLib import fontInfoAttributesVersion3
 
     ds = self.designspace
+
+    # unless the `fileName` custom parameter was explicitly set, do like Glyphs.app
+    # and concatenate the family, instance (style) names and "VF" to form a filename;
+    # the default 'Regular' is omitted by Glyphs.app.
+    # https://github.com/googlefonts/glyphsLib/issues/981
+    filename = instance.customParameters["fileName"]
+    if not filename:
+        filename = self.font.familyName
+        if instance.name != "Regular":
+            filename += "-" + instance.name
```

Comment:
Does GlyphsApp use instance name at all here?

Reply:
yes, it does, just tried again to confirm. It concatenates to the font familyName using a hyphen.

Reply:
Oh! I also just noticed that in addition to that, Glyphs.app also strips all the spaces from the generated filename. I guess we should do the same.